src/seedu/addressbook/AddressBook.java

```diff
@@ -411,6 +411,7 @@ private static String getMessageForInvalidCommandInput(String userCommand, Strin
     /**
      * Adds a person (specified by the command args) to the address book.
      * The entire command arguments string is treated as a string representation of the person to add.
+     * The command arguments string is added as upper case to allow for case-insensitive matching.
```

Comment:
I guess, you can just change `getPersonsWithNameContainingAnyKeyword` method in `executeFindPersons`